web/src/features/SurveyFormValidation.ts

```diff
@@ -0,0 +1,113 @@
+import { QuestionType } from '../types/request/Question';
```

Comment:
What is the purpose of this `features/` for? I see some util functions but what is the difference between this and  `utils/validate.ts` file? Less cohesion.

Reply:
I just want to manage validation logic separately, not managed on the page. 
I forgot utils/validate.ts is exist... I'll put them it together